pkg/awsManager/ec2Manager.go

```diff
@@ -0,0 +1,84 @@
+package awsManager
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	clientpkg "github.com/openshift/aws-account-shredder/pkg/aws"
+	"os"
+	"strings"
+)
+
+// this lists all the instances that are eligible for deletion based on the tags and stored them in instances to be deleted
+// this only creates an array of pointers and does not delete the instances
+func ListEc2InstancesForDeletion(client clientpkg.Client) []*string {
+
+	var EC2InstancesToBeDeleted []*string
+	token := ""
+	for {
+		ec2Descriptions, err := client.DescribeInstances(&ec2.DescribeInstancesInput{NextToken: aws.String(token)})
+		if err != nil {
+			fmt.Println("ERROR:", err)
+			os.Exit(1)
```

Comment:
Please remove this

Reply:
removed ,and removed the same in main,go as well